starter-code/pizza.js

```diff
@@ -1 +1,35 @@
 // Write your Pizza Builder JavaScript in this file.
+"use strict"
+
+$(".pep, .green-pepper, .mushroom").hide();
+
+$(".btn-pepperonni").click(function(){
+  $('.pep').toggle();
+});
+
+$(".btn-mushrooms").click(function() {
+  $(".mushroom").toggle();
+});
+
+$(".btn-green-peppers").click(function() {
+  $(".green-pepper").toggle();
+});
+
+$(".btn-crust").click(function() {
+  $(".crust-gluten-free").toggle();
+});
+
+$(".btn-sauce").click(function() {
+  $(".sauce-white").toggle();
```

Comment:
On the sauce and crust you needed to use toggleClass(); in order to not make the whole thing disappear